spec/features/hub/show_client_spec.rb

```diff
@@ -48,6 +48,18 @@
         expect(page).not_to have_css(".tax-return-inline-form")
       end
     end
+
+    context "for a client with an archived 2021 intake" do
+      before do
+        client.intake.destroy
+        @intake = create(:archived_2021_intake, client: client)
```

Comment:
Would this be tidier as a `let(:intake) { nil }` then `let(:archived_2021_intake) { create(:archived_2021_intake, client: client}`?

Reply:
Sure, let's try it